README.md

```diff
@@ -188,31 +188,31 @@ to get started with Vagrant, please see [CONTRIBUTING.md](CONTRIBUTING.md).*
 #### Publishing metrics to a file
 To start collecting Mesos metrics and publish them to a file, you'll need to perform the following steps.
 
-
-Start the Snap daemon in the background:
+Set up the [Snap framework](https://github.com/intelsdi-x/snap/blob/master/README.md#getting-started),
+in one terminal window, run `snapd` (in this case with logging set to 1, trust disabled and global configuration saved in snap-config-example.json):
```

Comment:
I don't think we need the part "in one terminal window" since the snap daemon is running as a background process in this example.